db/migrate/20230314144134_remove_not_null_constraint_from_data_services.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class RemoveNotNullConstraintFromDataServices < ActiveRecord::Migration[7.0]
+  def change
+    change_column_null :data_services, :organisation_id, true
```

Comment:
We should really get map the values in `organisation_id` column to `publisher_id` and then get rid of it, that means also updating the seed script and imports from NHS etc to save it as the publisher. Happy for us to leave that for a later day, in the interests of getting this API working!

Reply:
Likewise `name` -> `title`, and `url` -> `endpoint_url`

Reply:
that is the data_service, not the resource. publisher and title are on the resource